sceptre/cli/list.py

```diff
@@ -127,3 +126,39 @@ def list_change_sets(ctx, path, url):
 
     for response in responses:
         write(response, context.output_format)
+
+
+@list_group.group(name="local")
+def local_group():
+    """
+    Commands for listing attributes of local stacks.
+    """
+    pass
+
+
+@local_group.command(name="stacks")
+@click.argument("path")
+@click.pass_context
+@catch_exceptions
+def list_stacks(ctx, path):
+    """
+    List stack names and command paths for local stacks in a
+    stack group.
+    \f
```

Comment:
Help is cutting this off. please try to shorten the description slightly..

```
➜  sceptre list local --help
Usage: sceptre list local [OPTIONS] COMMAND [ARGS]...

  Commands for listing attributes of local stacks.

Options:
  --help  Show this message and exit.

Commands:
  stacks  List stack names and command paths for local stacks in a stack...
```

How about just `List sceptre stack config attributes`?